specification/columns/invoiceid.md

```diff
@@ -25,7 +26,7 @@ Invoice ID
 
 ## Description
 
-The provider-assigned identifier for an invoice encapsulating some or all *charges* in the corresponding billing period for a given billing account.
+The provider-assigned identifier for an invoice encapsulating all *charges* in the corresponding billing period for a given billing account.
```

Comment:
I don't think we can make this change. Billing periods may be covered by multiple invoices. Or at least, we'd need to say something like, "...that were billed on the invoice identified by the invoice ID." (But in a much more graceful way 🙃)

Reply:
I like this. Simple and safe.